src/AVKit/Enums.cs

```diff
@@ -24,4 +25,17 @@ public enum AVKitError : nint {
 		PictureInPictureStartFailed = -1001
 	}
 #endif
+
+#if TVOS
+	// this enum only exists for TVOS
+	[TV (10,1)]
+	public enum AVKitMetadataIdentifier : nint {
```

Comment:
Any reason to bind it as an enum ?

The usage is not very clear but the header seems to imply the constant will be used as a `NSString`...

e.g.

```
 @constant AVKitMetadataIdentifierExternalContentIdentifier
 @abstract AVKit will publish the value of this metadata item to MPNowPlayingInfoPropertyExternalContentIdentifier.
```

if only to allow comparison with the existing `MP*` constant.